frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.unit.spec.tsx

```diff
@@ -94,6 +94,16 @@ describe("QueryColumnPicker", () => {
     expect(onClose).toHaveBeenCalled();
   });
 
+  it("should render info icons", () => {
+    const { sampleColumnInfo } = setup();
+
+    expect(
+      within(
+        screen.getByLabelText(sampleColumnInfo.displayName),
+      ).getByLabelText("More info"),
+    ).toBeInTheDocument();
```

Comment:
We should also test the behavior on hover.

Reply:
That's already tested in the `ColumnInfoIcon` unit tests, do we need to retest it here?

Reply:
> That's already tested in the `ColumnInfoIcon` unit tests, do we need to retest it here?

Great! I think there's no need to retest it here :+1: